safeweb/http.go

```diff
@@ -144,6 +144,10 @@ type Config struct {
 	// BrowserMux when SecureContext is true.
 	// If empty, it defaults to max-age of 1 year.
 	StrictTransportSecurityOptions string
+
+	// HTTPServer, if specified, is the underlying http.Server that safeweb will
+	// use to serve requests. If nil, a new http.Server will be created.
```

Comment:
```suggestion
	// use to serve requests. If nil, a new http.Server will be created.
	// Do not use the Handler field of http.Server, as it will be ignored. Instead, set your handlers using APIMux and BrowserMux.
```